source/2019-03-30-the-ember-times-issue-91.md

```diff
@@ -10,15 +10,18 @@ responsive: true
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
 <!--alex ignore hangman-hangwoman-->
-Read some RFCs around Record Data Errors 📇 and extending `EmberObject` 💡, learn how to build Hangman with Ember Octane 🎬,
+Read some RFCs around Record Data Errors 📇 and extending `EmberObject` 💡, learn how to build Hangman with Ember Octane 🎬 and about the addon modules RFC, check out the "e"mpressive EmberWeekend podcast with @mansona, ...
 
 
 ...and read the exclusive interview with Ember contributor **@kategengler**.
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Configuring addon modules in Module Unification layout](https://github.com/ppcano/rfcs/blob/addon-module-config/text/0000-addon-module-config.md)
+
+A new RFC is out which proposes to provide an API for addons to configure their module types and collections for Module Unification apps. The reason for this is that at this moment Classic addons that require apps to locate classes into a non-default app folder do not work "out of the box" in an Module Unification application.
 
+If you're looking to adopt Module Unification in the future then make sure to have a [look at the RFC](https://github.com/emberjs/rfcs/pull/462) and share your thoughts.
```

Comment:
MU is having a re-think and probably will never look like the RFC for MU (See https://blog.emberjs.com/2019/03/11/update-on-module-unification-and-octane.html)

If we really want to include this, I think a warning pointing to the blog post on emberjs.com would be appropriate.

Reply:
Thank you for the heads up and the suggestion! I added a few suggestions for an update [here](https://github.com/ember-learn/ember-blog/pull/29#pullrequestreview-220690432) - what are your thoughts?